daprdocs/content/en/operations/performance-and-scalability/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+type: docs
+title: "Performance and Scalability"
+linkTitle: "Performance and Scalability"
+weight: 300
+description: "Benchmarks and guidelines"
+---
+Dapr allows developers to inject sidecars next to their apps that make it easy to write distribured applications. The Dapr APIs allow developers to manage state at scale, trigger their apps from external sources, fetch secrets, publish and consume messages, and call other apps in a secured manner with build in telemetry and resiliency.
```

Comment:
Delete this paragraph overview. we do not need to describe Dapr in every article

Reply:
Deleted